src/Razor/src/rzls/rzls.csproj

```diff
@@ -30,6 +30,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.AspNetCore.Razor.LanguageServer\Microsoft.AspNetCore.Razor.LanguageServer.csproj" />
+    <None Update=".\Targets\**\*.targets" CopyToOutputDirectory="PreserveNewest" />
+    <None Update=".\Targets\**\*.xaml" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
If it works, it works, but I wonder why `notice.txt`, below, is `Content`?

Reply:
Hmm... idk what the difference is. I can switch to content